website/src/Components.js

```diff
@@ -74,32 +74,23 @@ export function Selector() {
     view: vnode => {
       return [
         m('label.formlabel', { for: vnode.attrs.name }, vnode.attrs.text),
-        m('select', vnode.attrs, vnode.children.map(c => {
-          return m('option', {value: c}, c)
-        })),
+        m(
+          'select',
+          vnode.attrs,
+          vnode.children.map(c => {
+            return m('option', { value: c }, c)
+          })
+        ),
         m('br'),
       ]
-    }
+    },
   }
 }
 
-
-export function Section() {
-  return {
-    view: vnode => {
-      return m('section', vnode.attrs, vnode.children)
-    }
-  }
+export const Section = {
+  view: vnode => m('section', vnode.attrs, vnode.children),
 }
 
-export function Layout() {
-  return {
-    view: vnode => {
-      return [
-        m(Menu),
-        m(Section, vnode.attrs, vnode.children)
-      ]
-    }
-  }
+export const Layout = {
+  view: vnode => [m(Menu), m(Section, vnode.attrs, vnode.children)],
 }
```

Comment:
This is an excellent example of refactoring the unnecessary structure out of a component. Since this component doesn't need any internal state, it can become a POJO type component. 

Much easier to read!